shell/PTY4J.md

```diff
@@ -40,96 +40,53 @@ Also pty4j implements java interface for pty for windows, using [WinPty](https:/
 ## Dependencies
```

Comment:
can we delete the `HTERM.md` and `PTY4J.md`? I don't think we should copy the README when pulling a third-party dependency.